src/main/java/com/libertymutualgroup/herman/aws/cft/CftPush.java

```diff
@@ -75,6 +74,7 @@
     private static final String MAVEN_VERS = "maven.version";
     private static final int RANDOM_PASSWORD_LENGTH = 20;
     private static final int POLLING_INTERVAL_MS = 10000;
+    private static final List<String> CFT_FILE_NAMES = Arrays.asList("cft.template", "cft.yml", "cft.json");
```

Comment:
Unrelated change - Adding support for different filenames here too. In the future I'll try to split out PRs in a coherent way.